File: co.edu.uniandes.fixml.toCSharp/test/text2.xml/Qty.cs


// Class for tag Qty
class Qty {

	string _Typ;
	string _Long;
	string _Short;
	
	
	// default constructor
	
	public Qty ( ) {   
		this._Typ = "SOD";
		this._Long = "35";
		this._Short = "0";
		
		
	}
	
	public Qty ( string Typ, string Long, string Short ) {
		this._Typ = Typ;
		this._Long = Long;
		this._Short = Short;
		
	}
	
	
} 

File: co.edu.uniandes.fixml.toJava/test/text2.xml/Qty.java
package text2.xml;



// Class for tag Qty
public class Qty {

	String Typ;
	String Long;
	String Short;
	
	
	// default constructor
	public Qty ( String Typ, String Long, String Short ) {
		this.Typ = Typ;
		this.Long = Long;
		this.Short = Short;
		
	}
	
	
	public Qty ( ) {   
	
		this.Typ = "SOD";
		this.Long = "35";
		this.Short = "0";
		
	}
	
	
} 